src/components/AvatarPage/AvatarPage.tsx

```diff
@@ -27,9 +27,20 @@ export default class AvatarPage extends React.PureComponent<Props> {
               </Column>
               <Column align="right">
                 <Row className="actions">
-                  <Button basic className="create-item" onClick={() => onOpenModal('CreateItemModal')}>
-                    <Icon name="add-active" />
-                  </Button>
+                  <Dropdown
+                    trigger={
+                      <Button basic className="create-item">
+                        <Icon name="add-active" />
+                      </Button>
+                    }
+                    inline
+                    direction="left"
+                  >
+                    <Dropdown.Menu>
+                      <Dropdown.Item text={t('avatar_page.new_item')} onClick={() => onOpenModal('CreateItemModal')} />
+                      <Dropdown.Item text={t('avatar_page.new_collection')} onClick={() => onOpenModal('CreateCollectionModal')} />
```

Comment:
Is there a reason not to have this on instance methods ? (not blocking the merge, just asking)